api/envoy/api/v2/core/grpc_service.proto

```diff
@@ -88,12 +149,14 @@ message GrpcService {
     oneof credential_specifier {
       option (validate.required) = true;
 
-      // OAuth2 access token, see
-      // https://grpc.io/grpc/cpp/namespacegrpc.html#ad3a80da696ffdaea943f0f858d7a360d.
-      string access_token = 1;
-      // [#comment: TODO(htuch): other gRPC auth types, e.g. IAM credentials, JWT, etc.]
+      // Channel credentials
+      GoogleGrpc.ChannelCredentials google_channel_credentials = 1;
```

Comment:
I think it's a bit confusing to present the Google gRPC credentials as the general shared credentials. Either we get rid of Credentials here, and fold it into `GoogleGrpc`, or we make `Channel/CallCredentials` a shared concept, i.e. lift it out of `GoogleGrpc`.